src/CommonFiles/Oryx.ruleset

```diff
@@ -123,7 +123,7 @@
 
   <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.Features" RuleNamespace="Microsoft.CodeAnalysis.CSharp.Features">
     <!-- Requires that unused using directives are removed. -->
-    <Rule Id="IDE0005" Action="Error" />
+    <Rule Id="IDE0005" Action="Info" />
```

Comment:
nit: was this causing too many errors with the changes made in this PR?

Reply:
ah good catch, I was originally trying to keep errors from being thrown about the unused directives in the test projects, which ended up being solved by making the assets private for the analyzers: https://github.com/microsoft/Oryx/pull/1791/files#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16eR29-R31

so I'll revert this